### PR TITLE
chore: optimize `normalize` dependencies usage

### LINKED ISSUE #1494: Remove `@csstools/normalize.css` from foundations
- We have a dependency for foundations that [normalizes css](https://github.com/db-ui/mono/blob/main/packages/foundations/package.json#L31) but a lot of those normalizes are overwritten later
- We should remove the dependency and check which of the normalizes matters for us and which ones we should adapt with our own variables (tonality etc.)

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

closes #1494 

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Comment by mfranzke** [comment]
This is currently just removing normalize, but not replacing it with another solution that would ensure we stay up to date.

File: packages/foundations/scss/_normalize.scss
Comment (mfranzke) [inline packages/foundations/scss/_normalize.scss:1]: This is currently just removing normalize, but not replacing it with another solution that would ensure we stay up to date.

Comment (nmerget) [reply packages/foundations/scss/_normalize.scss:1]: The postcss plugin didn't work, maybe you can check it as well?
I don't like this hard dependency just for some CSS...

Comment (nmerget) [reply packages/foundations/scss/_normalize.scss:1]: fixed it

**Review by mfranzke — APPROVED** [review]

**Pull request merged**